extensions/amp-story/0.1/amp-story.js

```diff
@@ -717,8 +717,14 @@ export class AmpStory extends AMP.BaseElement {
    * @return {string} The URL of the background resource
```

Comment:
Seems like this should actually be `{?string}`

Reply:
Done